packages/mobile/src/services/analytics.ts

```diff
@@ -74,7 +76,8 @@ export const track = async ({ eventName, properties }: Track) => {
   const version = VersionNumber.appVersion
   const propertiesWithContext = {
     ...properties,
-    mobileClientVersion: version
+    mobileClientVersion: version,
+    mobileClientVersionInclOTA: versionInfo ?? 'unknown'
```

Comment:
Awesome! Could we also add the full version to https://github.com/AudiusProject/audius-client/blob/main/packages/common/src/services/remote-config/remote-config.ts#L226

then we can target feature flags to specific ota releases

Reply:
oo nice, yes!